package.json

```diff
@@ -7,8 +7,8 @@
     "serve": "webpack serve --hot --inline --mode development",
     "test": "npm run test-unit && npm run test-client",
     "test-unit": "mocha test/model-test.js",
-    "test-client": "mocha-chrome test/view-test.html",
-    "build": "webpack && webpack --config test/test.webpack.config.js"
+    "test-client": "mocha-chrome dist/view-test.html",
+    "build": "webpack --mode production && webpack --config test/test.webpack.config.js --mode development"
```

Comment:
For my own understanding, why is there a second run of Webpack w/ development settings?